rotkehlchen/chain/ethereum/modules/aave/v2/constants.py

```diff
@@ -8,3 +8,26 @@
     string_to_evm_address('0xEFFC18fC3b7eb8E676dac549E0c693ad50D1Ce31'),
     string_to_evm_address('0xcc9a0B7c43DC2a5F023Bb9b738E45B0Ef6B06E04'),
 )
+STK_AAVE: Final = string_to_evm_address('0x4da27a545c0c5B758a6BA100e3a049001de870f5')
+STAKED_AAVE: Final = b"l\x86\xf3\xfdQ\x18\xb3\xaa\x8b\xb4\xf3\x89\xa6\x17\x04m\xe0\xa3\xd3\xd4w\xde\x1a\x16s\xd2'\xf8\x02\xf6\x16\xdc"  # noqa: E501
+REWARDS_ABI = [
+    {
+        'inputs': [
+            {
+                'internalType': 'address',
+                'name': 'staker',
+                'type': 'address',
+            },
+        ],
+        'name': 'getTotalRewardsBalance',
+        'outputs': [
+            {
+                'internalType': 'uint256',
+                'name': '',
+                'type': 'uint256',
+            },
+        ],
+        'stateMutability': 'view',
+        'type': 'function',
+    },
+]
```

Comment:
why not add this abi to the globaldb? The idea is to reduce the number of constants in memory

Reply:
Added to globaldb